src/styles/stars.scss

```diff
@@ -0,0 +1,29 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  width: 200px;
+}
+
+.card__stars-list {
+  display: flex;
+}
+
+.card__star:not(:last-child) {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  padding-right: 4px;
```

Comment:
Using padding-right on .card__star:not(:last-child) might cause alignment issues if the stars are meant to be evenly spaced. Consider using margin-right instead to ensure consistent spacing between stars.